p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/Steps/EnterPhoneNumber/EnterPhoneNumberViewModel.swift

```diff
@@ -26,6 +26,7 @@ final class EnterPhoneNumberViewModel: BaseOTPViewModel {
     @Injected private var reachability: Reachability
     @Injected private var notificationService: NotificationService
     @Injected private var countriesAPI: CountriesAPI
+    @Injected private var helpCenterLauncher: HelpCenterLauncher
```

Comment:
RestoreCustomDelegatedCoordinator has openHelp() method, can you look at it? And all "openInfo" methods in RestoreCustomDelegatedCoordinator, RestoreSocialDelegatedCoordinator, RestoreWalletCoordinator, BindingPhoneNumberDelegatedCoordinator should open intercom I guess. Should they be deleted or be written somehow?

Reply:
@qwerkly Can you answer to @lisemyon's question?